SPECS-EXTENDED/aspell/aspell.spec

```
@@ -111,6 +114,9 @@ rm -f ${RPM_BUILD_ROOT}%{_infodir}/dir
 %{_mandir}/man1/pspell-config.1*
 
 %changelog
+* Fri Jun 17 2022 Jamie Magee <jamagee@microsoft.com> - 0.60.8-6
+- Switch from FTP to HTTPS source
+
```

```suggestion
- Switch from FTP to HTTPS source
- License verified

```

Went ahead and took a look at the licensing- seems that everything in this spec is consistent with the source. Once you add this line, I can approve and merge!